fetch.bs

```diff
@@ -2213,6 +2283,89 @@ clearly stipulates that <a>connections</a> are keyed on
 <!-- See https://github.com/whatwg/fetch/issues/114#issuecomment-143500095 for when we make
      WebSocket saner -->
 
+<p>The requirements for <dfn>recording connection timing info</dfn> given a <a for=/>connection</a>
+<var>connection</var> and its <a for=/>connection timing info</a> <var>timingInfo</var>, are as
+follows:
+<ul>
+ <li><p><var>timingInfo</var>'s <a for="connection timing info">domain lookup start time</a>
+ should be the <a for=/>unsafe shared current time</a> immediately before starting the domain
+ lookup, or beginning retrieval of the information from cache.
+
+ <li><p><var>timingInfo</var>'s <a for="connection timing info">domain lookup end time</a> should
+ be the <a for=/>unsafe shared current time</a> immediately after finishing the domain lookup, or
+ retrieving the information from cache.
+
+ <li><p><var>timingInfo</var>'s <a for="connection timing info">connection start time</a> should
+ be the <a for=/>unsafe shared current time</a> immediately before establishing the connection to
+ the server or proxy.
```

Comment:
We need to state what happens when a previous connection was reused

Reply:
It's specified: It's returned from the connection pool and already has an associated connection timing info, which gets clamped to fetch start in `clamp connection timing to fetch timing`. Something missing?